### ISSUE TITLE
CM database recipes for Croissant

### ISSUE THREAD

**Issue by gfursin** [mlcommons/croissant#604]
Hi,

We would like to add Croissant loader support for the [MLCommons CM automation recipes for MLPerf benchmarks](https://access.cknowledge.org/playground/?action=scripts#ai-ml-datasets).

I started adding Croissant examples to CM to be able to connect them with MLPerf and would like to ask if I can reuse [this Croissant example](https://github.com/mlcommons/croissant/pull/564/files) in this [CM script](https://github.com/mlcommons/ck/blob/master/cm-mlops/script/print-croissant-desc/code.py) ?

Do I need to add author/license? Both projects (CM and Croissant) are under MLCommons and Apache 2 license. 

Thank you!

CC @mkuchnik and @benjelloun .

**Comment by benjelloun** [comment]
Non-lawyer response: I think it's sufficient to say where the example comes from, e.g., by linking to the Croissant documentation page where it appears.

Does that make sense to you?

**Comment by gfursin** [comment]
Hi @benjelloun.
Sure, I already added a link to the code snippet itself:
```
# Taken from https://github.com/mlcommons/croissant/pull/564/files (@mkuchnik) 
```
However, I want to better acknowledge the original author and curious if there is a better/more official way to do so ... 
But I can just keep a link for now ...

**Comment by mkuchnik** [comment]
@gfursin Not a lawyer, but as far as credit goes the link seems reasonable to me.

**Comment by gfursin** [comment]
Cool! Thanks guys! I will close this ticket!
